game-core/src/main/java/games/strategy/engine/data/ModifiableProperty.java

```diff
@@ -0,0 +1,145 @@
+package games.strategy.engine.data;
+
+import java.util.function.Supplier;
+
+/**
+ * A wrapper interface to Bundle setters, getters and resetters of the same field.
+ *
+ * @param <T> The type of the field to set, get and reset.
+ */
+public interface ModifiableProperty<T> {
+
+  void setValue(String string) throws Exception;
```

Comment:
Yeah, I don't think we should be using a `throws Exception` specification in production code.  If you're trying to make this interface generic (i.e. not game parser specific), I would recommend creating a new type of checked exception specifically for this interface.